packages/app-accounts/test/enzyme.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+const Adapter = require('enzyme-adapter-react-16');
```

Comment:
We may want to move this file to the root under test - it will be used by other apps as well. (No need to add duplication in that case)